css/popup.css

```diff
@@ -34,7 +35,8 @@ body {
 #notifyHistories li .icon{
     width: 48px;
     height: 48px;
-    margin-right: 5px;
+    margin-right: 10px;
+    margin-left: -15px;
```

Comment:
Thank you, but design has broken a little because of this when `new` icon exists :sweat_drops:

## Before
![before](https://cloud.githubusercontent.com/assets/608755/6713340/011f6456-cdd5-11e4-8607-36ff952c3dc9.png)

## After (your change)
![after](https://cloud.githubusercontent.com/assets/608755/6713344/053de4c2-cdd5-11e4-924b-db74a674e7a6.png)

this margin is optimized for `new` icon